### PR TITLE
[scan] add reset_simulator option

### PR DESCRIPTION
**Pull request by wag-miles**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

File: scan/lib/scan/options.rb [hunk 1]
```diff
@@ -69,6 +69,13 @@ def self.available_options
                                      type: Boolean,
                                      optional: true),
 
+        # reset simulator
+        FastlaneCore::ConfigItem.new(key: :reset_simulator,
+                                     env_name: 'SCAN_RESET_SIMULATOR',
+                                     description: "Enabling this option will automatically erase the simulator before running the application",
+                                     default_value: false,
+                                     is_string: false),
```
Comment (joshdholtz) [inline scan/lib/scan/options.rb]: Can we make this change? 🙃 `is_string` will soon be gone
```suggestion
                                     type: Boolean),
```

**Review by joshdholtz — CHANGES_REQUESTED** [review]
Other than the one request for changing `is_string` to `type` this looks 💯

File: scan/lib/scan/options.rb (hunk 1 above)
Comment (wag-miles) [reply scan/lib/scan/options.rb]: Good to know. Yes please go ahead and change it. Thanks.

**Review by joshdholtz — APPROVED** [review]
This is great! Thanks for the contribution ❤️

**Pull request merged**